bool is_takeoff 
geometry_msgs/PoseStamped poseStamped 
--- 